Wings Panic - cannot unmarshal !!map into []string , Broken Config from panel

**Background (please complete the following information):**
* Panel or Daemon: Panel
* Version of Panel/Daemon: RC.6

**Describe the bug**
The panel creates a config with `allowed_mounts: {  }` and that causes wings to panic on boot
```
panic: yaml: unmarshal errors:
  line 17: cannot unmarshal !!map into []string

goroutine 1 [running]:
github.com/pterodactyl/wings/cmd.rootCmdRun(0x17ded20, 0xc0000794b0, 0x0, 0x1)
        /home/runner/work/wings/wings/cmd/root.go:110 +0x21a8
github.com/spf13/cobra.(*Command).execute(0x17ded20, 0xc00000e090, 0x1, 0x1, 0x17ded20, 0xc00000e090)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:846 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x17ded20, 0x0, 0xffffffff, 0xc000100058)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/pterodactyl/wings/cmd.Execute(...)
        /home/runner/work/wings/wings/cmd/root.go:355
main.main()
        /home/runner/work/wings/wings/wings.go:8 +0x2e
```
Changing it to `allowed_mounts: []` resolves this issue. Oddly enough using the tokened Auto-deploy this issue is never ran into 

**To Reproduce**
1. Copy config from panel to wings
2. Start wings

**Expected behavior**
The correct brackets to be used and the wings to start 👍

--- a/app/Models/Node.php
+++ b/app/Models/Node.php
@@ -181,7 +181,7 @@ class Node extends Model
      */
     public function getYamlConfiguration()
     {
-        return Yaml::dump($this->getConfiguration(), 4, 2);
+        return Yaml::dump($this->getConfiguration(), 4, 2, Yaml::DUMP_EMPTY_ARRAY_AS_SEQUENCE);
     }
 
     /**

